### DISCUSSION TITLE
Webhook event workflow_job, waiting action triggered twice when requiring approval.

### DISCUSSION THREAD

**Discussion by barendb** [community/community#63097] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

We have noticed that `workflow_job` webhook events with the `waiting` action, is triggered/fired twice when the job requires approval from a user to proceed.

**Some notes**
* Payload events are received milliseconds after one another
* Payloads for events are exactly the same
* Occurs for all repositories in our organization
* Does not occur for any of the other `workflow_job` events
* Only occurs on workflow jobs that requires approval from a user to proceed

**Steps to reproduce.**
1. Create a webhook listening to `workflow_job` events
2. Create a Github Action that requires approval to proceed

**Comment by johnolson2219** [comment]
could u please talk more about your problem?
